js/transForms.js

```diff
@@ -0,0 +1,30 @@
+// IE doesn't support locale number formats
+const decimalMark = (element, value) => [
+	element.type == 'number',
+	Number(value.replace(',', '.')),
+]
+
+/* We are simulating a recruitment,
+hence requesting salaries with the new size of the entreprise */
+const incrementEffectif = (element, value) => [
+	element.name == 'effectif_entreprise',
+	value + 1,
+]
+
+const dureeLegaleMensuelle = 151.66,
+	dureeLegaleHebdomadaire = 35
```

Comment:
I'm really uncomfortable about `const`. 
I think it would be better to use `let` in most cases and `const` for true constants, removing the need for `const MY_CONST` which means : here's my constant myConst, but since it's really a constant this time, unlike 99% of my variable declarations, I'll uppercase its name. 
:dizzy_face: